python/covid_graphs/covid_graphs/country_graph.py

```diff
@@ -6,14 +6,12 @@
 
 import click
 import click_pathlib
-from google.protobuf import text_format  # type: ignore
 from plotly.graph_objs import Figure, Layout, Scatter
 
-from . import formula, prediction_generator
+from . import predictions
 from .country_report import CountryReport, create_report
-from .formula import FittedFormula, TraceGenerator
-from .pb.atg_prediction_pb2 import CountryAtgParameters
-from .predictions import BK_20200329, BK_20200411, CountryPrediction, PredictionEvent, prediction_db
+from .formula import TraceGenerator
+from .predictions import BK_20200329, BK_20200411, CountryPrediction, PredictionEvent
 
 # Extend the predictions at least by 1/5th of the length of active cases.
 EXTENSION_RATIO = 0.2
```

Comment:
The `MAX_PEAK_DISTANCE` constant is now unused (I know we would eventually want to move it back)

Reply:
I'm not sure what is the suggestion here, the constant has moved :)

Reply:
It's not moved, it's still on line 20

Reply:
I have no idea how is this possible :D Not only I thought I moved it, but also I did not notice that it stayed here on github.